tests/mhd/alfvenwing_test/correct_output/run.log

```diff
@@ -0,0 +1,94 @@
+ Start integrating, print status every  3.00E+01 seconds
+   #        it        time          dt  wc-time(s)
+   #         0  0.0000E+00  5.9473E-04  4.3028E-03
+ -------------------------------------------------------------------------------
+ Saving visual data. Coordinate directions and variable names are:
+           1 X               
+           2 Y               
+           3 Z               
+           4 rho             rho             
+           5 v1              v1              
+           6 v2              v2              
+           7 v3              v3              
+           8 p               p               
+           9 b1              b1              
+          10 b2              b2              
+          11 b3              b3              
+ time =   5.9472817133443144E-004
+ -------------------------------------------------------------------------------
+   #        50  1.4759E-01  3.1301E-03  3.0172E+01
+   #        99  3.0096E-01  3.1301E-03  6.0598E+01
+   #       148  4.5434E-01  3.1301E-03  9.1038E+01
+   #       197  6.0772E-01  3.1301E-03  1.2136E+02
+   #       246  7.6110E-01  3.1301E-03  1.5158E+02
+   #       294  9.1134E-01  3.1301E-03  1.8169E+02
+   #       343  1.0626E+00  3.1301E-03  2.1206E+02
+   #       393  1.2191E+00  3.1301E-03  2.4271E+02
+   #       441  1.3694E+00  3.1301E-03  2.7287E+02
+   #       489  1.5196E+00  3.1301E-03  3.0332E+02
+   #       537  1.6699E+00  3.1301E-03  3.3344E+02
+   #       586  1.8232E+00  3.1301E-03  3.6375E+02
+   #       635  1.9766E+00  3.1301E-03  3.9402E+02
+   #       683  2.1252E+00  3.1301E-03  4.2442E+02
+   #       731  2.2755E+00  3.1301E-03  4.5492E+02
+   #       779  2.4257E+00  3.1301E-03  4.8553E+02
+   #       827  2.5759E+00  3.1301E-03  5.1599E+02
+   #       875  2.7262E+00  3.1301E-03  5.4614E+02
+   #       923  2.8764E+00  3.1301E-03  5.7633E+02
+   #       971  3.0250E+00  3.1301E-03  6.0670E+02
+   #      1019  3.1753E+00  3.1301E-03  6.3703E+02
+   #      1066  3.3224E+00  3.1301E-03  6.6704E+02
+   #      1114  3.4727E+00  3.1301E-03  6.9740E+02
+   #      1162  3.6229E+00  3.1301E-03  7.2790E+02
+   #      1210  3.7731E+00  3.1301E-03  7.5818E+02
+   #      1258  3.9234E+00  3.1301E-03  7.8863E+02
+ Total timeloop took        :      804.428 sec
+ Time spent on Regrid+Update:       27.686 sec
+                  Percentage:         3.44 %
+ Time spent on IO in loop   :        1.217 sec
+                  Percentage:         0.15 %
+ Time spent on BC           :      348.996 sec
+                  Percentage:        43.38 %
+ Time spent on run          :      803.211 sec
+ Cells_updated / cpu / sec  :    7.349E+04
+ Total time spent on IO     :        1.378 sec
+ Total timeintegration took :      804.589 sec
+   #      1283   4.000E+00   0.000E+00   8.044E+02
+ -------------------------------------------------------------------------------
+ Finished AMRVAC in :           804.704 sec
```

Comment:
All MHD tests combined take approximately 45 minutes to complete, so an additional 12 minutes for a single 3D test is maybe a bit much... Is it possible to lower the runtime a bit, maybe decrease resolution/stopping time?